client/src/index.js

```diff
@@ -10,109 +10,146 @@ import InterpreterSignUpForm from "./components/InterpreterSignUpForm";
-          <Link className="nav-link" to="/">
-            <Menu.Item name="editorials" active={true}>
-              Home
-            </Menu.Item>
-          </Link>
-          {isLoggedIn() ? (
-            <Link className="nav-link" to="/Dashboard">
-              <Menu.Item name="editorials" active={true}>
-                Dashboard
-              </Menu.Item>
-            </Link>
-          ) : null}
-          <Link className="nav-link" to="/about">
-            <Menu.Item name="editorials" active={true}>
-              About
-            </Menu.Item>
-          </Link>
-
-          <Link className="nav-link" to="/status">
-            <Menu.Item name="editorials" active={true}>
-              Status
-            </Menu.Item>
-          </Link>
+  handleItemClick = (e, { name }) => this.setState({ activeItem: name });
 
-          {isLoggedIn() ? (
-            <Menu.Menu position="right">
-              <Link className="nav-link" to="/add-document/">
-                <Menu.Item name="editorials" active={true}>
-                  Add document
-                </Menu.Item>
-              </Link>
-
-              <Menu.Item active={true} onClick={logout}>
-                Logout
-              </Menu.Item>
-            </Menu.Menu>
-          ) : (
-            <Menu.Menu>
-              <Link className="nav-link" to="/sign-up-user">
-                <Menu.Item name="editorials" active={true}>
-                  User Sign Up
-                </Menu.Item>
-              </Link>
+  logout = () => {
+    sessionStorage.clear();
+    window.location.href = "/";
+  };
 
-              <Link className="nav-link" to="/login">
-                <Menu.Item name="editorials" active={true}>
-                  Login
-                </Menu.Item>
-              </Link>
-
-              <Link className="nav-link" to="/sign-up-interpreter">
-                <Menu.Item name="editorials" active={true}>
-                  Become Interpreter
-                </Menu.Item>
-              </Link>
-            </Menu.Menu>
-          )}
-        </Menu>
+  render() {
+    const { activeItem } = this.state;
+    return (
+      <Router>
         <div>
-          <Route path="/" exact component={Home} />
-          <Route path="/about/" component={About} />
-          <Route path="/status/" component={Status} />
-          <Route path="/sign-up-user/" component={UserSignUpForm} />
-          <Route path="/login/" component={Login} />
-          <Route
-            path="/sign-up-interpreter/"
-            component={InterpreterSignUpForm}
-          />
-          {isLoggedIn() ? (
-            <div>
-              <Route path="/add-document/" component={AddDocumentForm} />
-              <Route path="/dashboard" component={Dashboard} />
-              {isUser() ? null : (
-                <Route
-                  path="/add-document-translation/:documentId"
-                  component={AddTranslationForm}
+          <Menu borderless inverted color="grey">
```

Comment:
How about using the color black here? So the navbar stands out better from the rest of the page and is more visible, what do you think?

Reply:
Maybe [https://ianstormtaylor.com/design-tip-never-use-black/](avoid pure black) in favour of a dark grey? I read about this recently (via https://refactoringui.com/) and have noticed since that a lot of sites do actually avoid it in practice (even the text here isn't pure black).

Reply:
I agree @tekul , I think in Semantic UI, what they call "black" is actually not black, it's a very dark grey actually.